src/DreamCommerce/Component/Common/Model/ArrayableTrait.php

```diff
@@ -10,8 +10,9 @@
 
 namespace DreamCommerce\Component\Common\Model;
 
-use ArrayAccess;
-use DateTime;
+use \ArrayAccess;
+use \DateTime;
+use Sylius\Component\Resource\Model\ResourceInterface;
```

Comment:
Specjalnie wrzuciłem to w ciało metody bo być może aplikacje nie będzie korzystać z syliusa